tests/static_role_permissions/test_static_role_permissions.py

```diff
@@ -0,0 +1,110 @@
+from unittest import mock
+
+from django.contrib.auth.models import AnonymousUser, User
+from django.test import TestCase, override_settings
+
+from ambient_toolbox.static_role_permissions.permissions import load_static_role_permissions
+
+
+@override_settings(
+    AUTHENTICATION_BACKENDS=[
+        "ambient_toolbox.static_role_permissions.auth_backend.StaticRolePermissionBackend",
+    ]
+)
+class StatiRolePermissionTests(TestCase):
+    @override_settings(STATIC_ROLE_PERMISSIONS_PATH="tests.static_role_permissions.dummy_permissions.PERMISSIONS_DICT")
+    def test_load_permissions_from_settings_path(self):
+        permissions = load_static_role_permissions()
+        self.assertEqual(
+            {"role_1": {"auth.add_user", "auth.view_user"}, "role_2": {}},
+            permissions,
+        )
+
+    def test_load_permissions_from_settings_path_without_path_set(self):
+        with self.assertRaisesMessage(AssertionError, "STATIC_ROLE_PERMISSIONS_PATH is not set in settings.py"):
+            load_static_role_permissions()
+
+    @override_settings(STATIC_ROLE_PERMISSIONS_PATH="tests.static_role_permissions.dummy_permissions.PERMISSIONS_LIST")
```

Comment:
question: would it be easier to remove the variable and use a `return_value=[]` here instead?

Reply:
yes, but here I exactly want to test loading the permissions from the path variable.

Reply:
fair enough